laskyawm-bar.js

```diff
@@ -1,5 +1,24 @@
 /* This file is licensed under the Affero General Public License. */
 
-var bar = document.createElement('div');
-bar.id = 'bar';
-document.body.appendChild(bar);
\ No newline at end of file
+(function(){
+	// Add namespace
+	window.bar = {};
+	
+	// Add bar
+	var barElement = document.createElement('div');
+	barElement.id = 'bar';
+	document.body.appendChild(barElement);
+
+	// Add icon panel
+	var barIcons = document.createElement('div');
+	barIcons.id = 'bar-icons';
+	barElement.appendChild(barIcons);
+	
+	bar.addItem = function (node) {
+		$(barIcons).prepend(node);		
+	}
```

Comment:
Could you run this through the style checker? Instructions here: https://github.com/airbornos/airbornos/blob/4388129/CONTRIBUTING.md, and then `git commit --amend`